natlas-server/app/models/scope_item.py

```diff
@@ -62,8 +62,6 @@ def parse_import_line(line):
             ip = line
             tags = []
 
-        if "/" not in ip:
-            ip = ip + "/32"
```

Comment:
This is a good remove. Not only was this a bad assumption, anytime we're scattering the same logic all over the place, it's an indication that we need to consolidate it.